### DISCUSSION TITLE
Call VSG from java in Android

### DISCUSSION THREAD

**Discussion by HUSTWillHe** [vsg-dev/VulkanSceneGraph#1060] [Q&A]
I hope to use JNI to call VSG in Android, instead of using native activity, I'm not quiet sure how to achieve this. Is there an example? Or could you give me some hints?  
I can only find this (https://github.com/vsg-dev/vsgExamples/tree/master/examples/platform/vsgandroidnative)  
Thank you!

**Comment by robertosfield** [comment]
The vsgandroidnative example is the only Android example we currently have, but I'm not the author of this particular example and don't have Android expertise so can't personally provide guidance on this side.  Hopefully others in the community can provide insights.

Failing this you'll need to try things out and let us know what works or doesn't work and we can try to figure out a path forward. Links to information or examples that illustrate what you are trying to achieve would be helpful.
